test/rates_test.rb

```diff
@@ -221,6 +231,29 @@ def test_invalid_currency_code_fails_with_rate_not_found_error
     end
   end
 
+  def test_exchange_rate_set_not_cached
+    cache = OpenExchangeRates::Cache::MemoryCache.new
+    stub(cache).get { nil }
+    stub(cache).set { open_asset('2012-05-10.json') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.